### PR TITLE
Move AzureDevOpsClient to Microsoft.DotNet.AzureDevOps.Authentication

### PR DESCRIPTION
**Pull request by adiaaida**
We would like to be able to share this code with Helix services, so this change moves it to a shared library.

### CODE REVIEW

File: src/Maestro/MaestroApplication/ApplicationPackageRoot/ApplicationManifest.xml [hunk 1]
```diff
@@ -132,6 +132,6 @@
     <DefaultRunAsPolicy UserRef="MaestroUser" />
   </Policies>
   <Certificates>
-    <EndpointCertificate X509FindType="FindBySubjectName" X509FindValue="[WebCertName]" Name="WebSslCert" />
```
Comment (garath) [inline src/Maestro/MaestroApplication/ApplicationPackageRoot/ApplicationManifest.xml:135]: This is one of those changes that VS makes, right? I'm not sure if it should be included here.

**Review by garath — APPROVED** [review]

File: src/Maestro/MaestroApplication/ApplicationPackageRoot/ApplicationManifest.xml (hunk 1 above)
Comment (adiaaida) [reply src/Maestro/MaestroApplication/ApplicationPackageRoot/ApplicationManifest.xml:135]: Stupid vs. I forgot to und that. Thanks :)

**Review by ChadNedzlek — CHANGES_REQUESTED** [review]
This isn't an authentication thing, so this isn't the right place for this.  Also, we already do a lot of Azure dev ops stuff in Helix, but using their SDK, rather than this weird zombie thing, so I'm not sure why we want to share it?

**Comment by adiaaida** [comment]
Dropping this PR. Chad pointed me to VssConnectionProvider, which I will be using instead.

**Pull request closed without merging**